lib/views/home_content/challenge/challenge_list.dart

```diff
@@ -11,16 +11,22 @@ class ChallengeList extends ConsumerWidget {
   Widget build(BuildContext context, WidgetRef ref) {
     final asyncChallenges = ref.watch(challengeProvider);
 
+    Widget emptyChallenge = const Center(
+      child: Text("No challenges available here"),
```

Comment:
Small typo ;)
```suggestion
      child: Text("No challenges available here"),
```

Reply:
You are completely right ! Fixed in: [3aef58c](https://github.com/ProximaEPFL/proxima/pull/215/commits/3aef58ca7c73026cd665adc21e76f62341567fef)